modin/core/io/text/text_file_dispatcher.py

```diff
@@ -48,6 +49,7 @@ class TextFileDispatcher(FileDispatcher):
     read_callback = None
 
     @classmethod
+    @logger_decorator("PANDAS-API", "TextFileDispatcher.get_path_or_buffer", "INFO")
```

Comment:
Is it possible to use a class decorator instead of all these function-level decorators?

Reply:
I agree, it just seems to me that all methods of `TextFileDispatcher` class are wrapped in decorators. Why not replace it with a class-level decorator in this case?

Reply:
@anmyachev it was not possible in this case due to the way that `TextFileDispatcher` is architected (it is a mix-in rather than a directly created/called class). We tried multiple ways, and this was the only way we could get it to work.